nearlib/test/integration.test.js

```diff
@@ -21,6 +25,73 @@ test('test creating default config', async () => {
     Near.createDefaultConfig();
 });
 
+describe("dev connect", () => {
+    let deps;
+    beforeEach(async () => {
+        const keyStore = new InMemoryKeyStore();
+        const storage = createFakestorage();   
+        deps = {
+            keyStore,
+            storage,
+            createAccount: (async (newAccountName, newAccountPublicKey) => {
+                const createAccountResponse = await account.createAccount(newAccountName, newAccountPublicKey, 1, aliceAccountName);
+                await nearjs.waitForTransactionResult(createAccountResponse);
+            })
+        };
+    });
+
+    test('test dev connect with no account creates a new account', async () => {
+        await dev.connect({deps});
+        expect(Object.keys(deps.keyStore.keys).length).toEqual(1);
+        const newAccountId = Object.keys(deps.keyStore.keys)[0];
+        const viewAccountResponse = await account.viewAccount(newAccountId);
+        const newAccountKeyPair = await deps.keyStore.getKey(newAccountId);
+        expect(newAccountKeyPair).toBeTruthy();
+        const expectedAccount = {
+            nonce: 0,
+            account_id: newAccountId,
+            amount: 1,
+            code_hash: newAccountCodeHash,
+            stake: 0,
+        };
+        expect(viewAccountResponse).toEqual(expectedAccount);
+        expect(deps.storage.getItem(storageAccountIdKey)).toEqual(newAccountId);
+    });
+
+    test('test dev connect with invalid account in storage creates a new account', async () => {
+        // set up invalid account id in local storage
+        deps.storage.setItem(storageAccountIdKey, await generateUniqueString("invalid"));
+        await dev.connect({deps});
+        expect(Object.keys(deps.keyStore.keys).length).toEqual(1);
+        const newAccountId = Object.keys(deps.keyStore.keys)[0];
+        const viewAccountResponse = await account.viewAccount(newAccountId);
+        const newAccountKeyPair = await deps.keyStore.getKey(newAccountId);
+        expect(newAccountKeyPair).toBeTruthy();
+        const expectedAccount = {
+            nonce: 0,
+            account_id: newAccountId,
+            amount: 1,
+            code_hash: newAccountCodeHash,
+            stake: 0,
+        };
+        expect(viewAccountResponse).toEqual(expectedAccount);
+        expect(deps.storage.getItem(storageAccountIdKey)).toEqual(newAccountId);
+    });
+
+    test('test dev connect with valid account but no keys', async () => {
+        // setup: connect with dev, but rmemove keys afterwards!
+        deps.storage.setItem(storageAccountIdKey, await generateUniqueString("invalid"));
+        const nearNoAccount = await dev.connect({deps});
```

Comment:
please use `npm run lint` and `npm run fix` to get rid of stuff like unused variables.